buildpipeline/Core-Setup-RHEL7-x64.json

```diff
@@ -22,6 +22,29 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
+      "displayName": "Initialize Docker",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "6c731c3c-3c68-459a-a5c9-bde6e6595b5b",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        // init-docker.sh was manually copied from dotnet/buildtools, as opposed to restoring buildtools package.
```

Comment:
Right...and I am assuming they get lost if you were to import the definition and re-export it.  Therefore comments like this are going to be a pain in the neck to maintain/keep when using our tooling to import/export the definitions when making changes.